crates/ruma-events/src/rtc/decline.rs

```diff
@@ -0,0 +1,84 @@
+//! Type for the MatrixRTC decline event ([MSC4310]).
+//!
+//! Unstable: `org.matrix.msc4310.rtc.decline`
+//!
+//! This event is sent as a reference relation to an `m.rtc.notification` event.
+//!
+//! [MSC4075]: https://github.com/matrix-org/matrix-spec-proposals/pull/4075
```

Comment:
Wrong MSC

Reply:
Oh noo. Copy buffer confusin. Thank you for pinging me on it.

(I ameded the change into the last commit)